target/linux/armsr/armv8/config-6.1

```diff
@@ -603,7 +608,6 @@ CONFIG_RANDOMIZE_BASE=y
 CONFIG_RANDOMIZE_MODULE_REGION_FULL=y
 CONFIG_RANDSTRUCT_NONE=y
 CONFIG_RELOCATABLE=y
-# CONFIG_RAVB is not set
```

Comment:
You should not _need_ to change this.
When OpenWrt is built for this target, with OpenWrt config including the new package `CONFIG_PACKAGE_kmod-renesas-net=y`, kernel config should be automatically modified.

Reply:
Hi @john-tho 
Thanks. I've reverted this change.